hubot-web-screenshot/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "hubot-web-screenshot",
+  "version": "1.0.0",
+  "description": "Takes a screenshot of a website and posts the resulting image. Supports various authentication methods.",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha",
+    "debug-test": "mocha --debug-brk --inspect"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rheobot-team/rheobot.git"
```

Comment:
git+?

Reply:
that was auto geneated by npm init. I guess that must be the convention if you support clone via SSH and also HTTPS?